### PR TITLE
HIPP-1628: Diagnostic scope view

### PR DESCRIPTION
**Pull request by PaulCDurham**
https://jira.tools.tax.service.gov.uk/browse/HIPP-1628

### CODE REVIEW

File: app/services/ApiHubService.scala [hunk 1]
```diff
@@ -214,4 +214,8 @@ class ApiHubService @Inject()(
     applicationsConnector.listEgressGateways()
   }
 
+  def fetchAllScopes(applicationId: String)(implicit hc: HeaderCarrier): Future[Option[Seq[CredentialScopes]]] = {
+    applicationsConnector.fetchALlScopes(applicationId)
```
Comment (victorarbuesmallada) [inline app/services/ApiHubService.scala:218]:
```suggestion
    applicationsConnector.fetchAllScopes(applicationId)
```

File: test/viewmodels/application/ApplicationNavItemsSpec.scala [hunk 2]
```diff
@@ -133,7 +141,9 @@ class ApplicationNavItemsSpec extends SpecBase with Matchers with TestHelpers wi
       }
     }
```
Comment (victorarbuesmallada) [inline test/viewmodels/application/ApplicationNavItemsSpec.scala:143]: Should there be a test case for the scenario in which the `ManageTeamMembers` nav item is hidden when the team is not migrated:
https://github.com/hmrc/api-hub-frontend/pull/434/files#diff-fc66007bfa67e5e66f02aa4442334c9a89c7488eb2b6d8964244a780012c3ae5R63

**Review by victorarbuesmallada — APPROVED** [review]

File: test/viewmodels/application/ApplicationNavItemsSpec.scala (hunk 2 above)
Comment (PaulCDurham) [reply test/viewmodels/application/ApplicationNavItemsSpec.scala:143]: That nav item should not be hidden when the team is not migrated so no I don't think we need that test case.

File: app/connectors/ApplicationsConnector.scala [hunk 3]
```diff
@@ -598,4 +598,17 @@ class ApplicationsConnector @Inject()(
       .setHeader(AUTHORIZATION -> clientAuthToken)
       .execute[Seq[EgressGateway]]
   }
+
+  def fetchALlScopes(applicationId: String)(implicit hc: HeaderCarrier): Future[Option[Seq[CredentialScopes]]] = {
```
[the remark this answers is not in the document]
Comment (PaulCDurham) [reply app/connectors/ApplicationsConnector.scala]: Typo, I'll fix that

**Review by victorarbuesmallada — APPROVED** [review]

**Pull request merged**